packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx

```diff
@@ -87,6 +87,13 @@ const VisitSummary: React.FC<VisitSummaryProps> = ({ encounters, patientUuid })
     encounters,
   ]);
 
+  const testsFilter = useCallback(() => {
```

Comment:
Does this actually work when tests are present? Normally, with `useCallback`, I'd expect to see the callback invoked somewhere, but the only use I see is on line 122 where we check the `length` property of the function, which should be [the number of arguments it takes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/length).

Reply:
@icrc-jofrancisco were you able to work on this change request?

Reply:
> 

@denniskigen  should be ready for merge now. Thanks.